src/test/typescript-service-helpers.ts

```diff
@@ -3309,8 +3309,8 @@ export function describeTypeScriptService(
                             fileName: toUnixPath(uri2path(rootUri + 'a.ts')), // path only used by TS service
                             textChanges: [
                                 {
-                                    span: { start: 49, length: 13 },
-                                    newText: '\t\tthis.missingThis',
+                                    span: { start: 51, length: 11 },
+                                    newText: 'this.missingThis',
```

Comment:
Seems like tabs are not anymore captured

Reply:
That's fine since `start` is excluding the tabs